### PR TITLE
Small updates

### PR DESCRIPTION
**Pull request by Coder-256**
Enable Travis CI cache, disable [deprecated container-based builds](https://docs.travis-ci.com/user/reference/trusty/#container-based-infrastructure), and add `red.png` to `.gitignore` (output from `cargo run --example capture`).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 language: rust
-sudo: false
+cache: cargo
```
Comment (kvark) [inline .travis.yml]: are you sure that's a good idea? My understanding was that the cache is huge in size and the costs of keeping it are not always justified.

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 language: rust
-sudo: false
```
Comment (kvark) [inline .travis.yml]: TIL!

File: .travis.yml (hunk 1 above)
Comment (Coder-256) [reply .travis.yml]: I'm not sure. I hadn't heard that, I had just assumed it was better from experience with other languages. It looks like there is a mix of projects that use `cache: cargo`, some that cache only `$HOME/.cargo/` and some that don't use caching at all, I'm not sure which is better.

**Comment by kvark** [comment]
alright, let's keep an eye on it
bors r+

File: .travis.yml (hunk 1 above)
Comment (rukai) [reply .travis.yml]: The problem with caching is cargo never cleans up old files built from old rustc versions or old dependencies.

**Pull request merged**